microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,233 @@
+
+To detect when your app runs in the sidebar, do one of the following:
+
+* On your web server, read the `Sec-CH-UA` HTTPS request header and look for the `Edge Side Panel` brand. For example:
+    
+    ```https
+    Sec-CH-UA: "Microsoft Edge";v="112", "Edge Side Panel";v="1", "Placeholder;Browser Brand";v="99"
+    ```
+
+* In the browser, use the `navigator.userAgentData` JavaScript API and read the value of the `brands` property. For example:
+
+    ```javascript
+    const brands = navigator.userAgentData.brands;
+    const sidebarBrandInfo = brands.find(b => b.brand === "Edge Side Panel");
+    if (sidebarBrandInfo) {
+      console.log(sidebarBrandInfo); // { brand: "Edge Side Panel", version: "1" }
+    } else {
+      console.log("App is not running in the Microsoft Edge sidebar");
+    }
+    ```
+
+You can also use the User Agent string to know whether your app runs in the sidebar or not.
+
+**Note**: we strongly recommend to use User Agent Client Hints instead. Using the User Agent string is an outdated mechanism for doing browser detection that has a long history of website compatibility issues.
+
+If you can't use User Agent Client Hints in your app, look for `EdgeSidePanel` in the User Agent string. Here are some examples of User Agent strings containing the `EdgeSidePanel` value:
+
+```https
+User-Agent: ... (EdgeSidePanel)...
+User-Agent: ... (..., EdgeSidePanel)...
+User-Agent: ... (EdgeSidePanel, ...)...
+```
+
+#### Relationship with your mobile app
+
+You might have created two variants of your app:
+
+* One variant when the app is used on a desktop device.
+* Another variant when the app is used on a mobile device.
+
+In this scenario, the desktop variant is used when your app is pinned to the sidebar. By default, apps in the sidebar receive the Microsoft Edge desktop User Agent Client Hint:
+
+```https
+Sec-CH-UA-Mobile: ?0
+```
+
+As explained in [Adapt your app to the minimum sidebar width](#adapt-your-app-to-the-minimum-sidebar-width), we recommend making your app's layout responsive so that it can adapt to multiple form factors using a single code base.
+
+You can, however, use the `Edge Side Panel` User Agent Client Hint brand to reuse your app's mobile variant in the Microsoft Edge sidebar.
+
+Follow these recommendations when using a mobile-variant of your app in the sidebar:
+
+* Remove all "Open in App" messages instructing users to download your app from an app store.
+* Test the accessibility and usability of your app with all input methods: mouse, keyboard, and touch. To learn about testing the accessibility of your app, see [Overview of accessibility testing using DevTools](../../devtools-guide-chromium/accessibility/accessibility-testing-in-devtools.md).
+
+
+<!-- ====================================================================== -->
```

Comment:
Can we add a section to get feedback from developers on Sidebar features they'd like to add/see for PWAs? Not sure we have a GitHub for PWA specific developer feedback. Something that was discussed, but we'd like feedback on before reconsidering was the ability to specify a start_url in the edge_side_panel manifest section.

Reply:
https://github.com/MicrosoftDocs/edge-developer/issues could be a reasonable place to direct Issues. Thanks for the pointer Hoch!

Reply:
We have the MSEdgeExplainers repo that can be used for user feedback about new features. I'd prefer if we could create a new explainer there to engage with people and collect their feedback rather than use the edge-developer docs repo.

Reply:
Agreed. Adolf created this explainer now.: https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/SidePanel/explainer.md.